changelogs/fragments/246-mariadb-integration-ci.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - CI - add ci tests for mariadb (https://github.com/ansible-collections/community.mysql/issues/238 https://github.com/ansible-collections/community.mysql/pull/246).
```

```suggestion
  - CI - add ci tests for mariadb (https://github.com/ansible-collections/community.mysql/issues/238 https://github.com/ansible-collections/community.mysql/pull/246).
```
The changelog fragment is not needed in this case as well as for all other changes that don't touch the code like doc ones for example

---

@rsicart could you please remove the fragment? Then we can merge the PR